x/supplier/client/cli/query_claim_test.go

```diff
@@ -109,48 +231,107 @@ func TestListClaim(t *testing.T) {
 		}
 		return args
 	}
+
 	t.Run("ByOffset", func(t *testing.T) {
 		step := 2
-		for i := 0; i < len(objs); i += step {
-			args := request(nil, uint64(i), uint64(step), false)
-			out, err := clitestutil.ExecTestCLICmd(ctx, cli.CmdListClaim(), args)
+		for i := 0; i < totalClaims; i += step {
+			args := prepareArgs(nil, uint64(i), uint64(step), false)
+			out, err := clitestutil.ExecTestCLICmd(ctx, cli.CmdListClaims(), args)
 			require.NoError(t, err)
+
 			var resp types.QueryAllClaimsResponse
 			require.NoError(t, net.Config.Codec.UnmarshalJSON(out.Bytes(), &resp))
+
 			require.LessOrEqual(t, len(resp.Claim), step)
 			require.Subset(t,
-				nullify.Fill(objs),
+				nullify.Fill(claims),
 				nullify.Fill(resp.Claim),
 			)
 		}
 	})
+
 	t.Run("ByKey", func(t *testing.T) {
 		step := 2
 		var next []byte
-		for i := 0; i < len(objs); i += step {
-			args := request(next, 0, uint64(step), false)
-			out, err := clitestutil.ExecTestCLICmd(ctx, cli.CmdListClaim(), args)
+		for i := 0; i < totalClaims; i += step {
+			args := prepareArgs(next, 0, uint64(step), false)
+			out, err := clitestutil.ExecTestCLICmd(ctx, cli.CmdListClaims(), args)
 			require.NoError(t, err)
+
 			var resp types.QueryAllClaimsResponse
 			require.NoError(t, net.Config.Codec.UnmarshalJSON(out.Bytes(), &resp))
+
 			require.LessOrEqual(t, len(resp.Claim), step)
 			require.Subset(t,
-				nullify.Fill(objs),
+				nullify.Fill(claims),
 				nullify.Fill(resp.Claim),
 			)
 			next = resp.Pagination.NextKey
 		}
 	})
-	t.Run("Total", func(t *testing.T) {
-		args := request(nil, 0, uint64(len(objs)), true)
-		out, err := clitestutil.ExecTestCLICmd(ctx, cli.CmdListClaim(), args)
+
+	t.Run("ByAddress", func(t *testing.T) {
+		args := prepareArgs(nil, 0, uint64(totalClaims), true)
+		args = append(args, fmt.Sprintf("--%s=%s", cli.FlagSupplierAddress, claims[0].SupplierAddress))
+
+		out, err := clitestutil.ExecTestCLICmd(ctx, cli.CmdListClaims(), args)
 		require.NoError(t, err)
+
 		var resp types.QueryAllClaimsResponse
 		require.NoError(t, net.Config.Codec.UnmarshalJSON(out.Bytes(), &resp))
+
+		require.Equal(t, numSessions, int(resp.Pagination.Total))
+		// require.ElementsMatch(t,
```

Comment:
is this causing issues?

Reply:
It's not, but ty for the reminder. Just added these back in!